packages/accordion/README.md

```diff
@@ -14,15 +14,14 @@ npm install @zendeskgarden/container-accordion
 
 ## Usage
 
-For live examples check out our
-[storybook](https://zendeskgarden.github.io/react-containers/?path=/story/accordion-container--useaccordion).
+Check out [storybook](https://zendeskgarden.github.io/react-containers) for live examples.
 
 ### useAccordion
 
 The `useAccordion` hook manages toggle state and required accessibility
 attributes for a group of sections.
 
-```jsx static
```

Comment:
These tags are leftover from styleguidist usage and prevent proper codeblock display in storybook.